### DISCUSSION TITLE
Release date for Version 4.0.0?

### DISCUSSION THREAD

**Discussion by blazing-panda** [passwordless-lib/fido2-net-lib#542] [Q&A] [answered]
Hey I am currently using this library to add Fido2 support to our Asp.Net Application, seems pretty straight forward 👍🏼 

I am currently using the `4.0.0.13 Beta` packages and was wondering if there is already a planned release date for version `4.0.0`?

I have already noticed some minor differences between the current master and `4.0.0.13 Beta` packages :)

**Comment by abergs** [comment, marked as answer]
We're trying to land #531 before 4.0 stable release, and perhaps some other changes of the API signature (but no behavioural changes)

**Reply by msheeri** [reply]
Hey, we are currently evaluating this library for our backend to integrate with webauthn. Would really appreciate if you can give some idea on when the next stable release is due for this?

**Reply by abergs** [reply]
I can't deliver a deadline for an open source project that relies on multiple contributors aligning their work. If you're looking for a more reliable partner I'd suggest looking at Bitwardens passwordless.dev offering which is really good (caveat, I work for Bitwarden)

**Reply by changeschung** [reply]
Isn't that identity theft cause i almost lost everything even my binance take your money give my account back through my ads you guys mafe almost 154k$ in a week this is all about or else?

**Comment by abergs** [comment]
4.0.0 has been released: https://github.com/passwordless-lib/fido2-net-lib/discussions/600
